js/build.ts

```diff
@@ -4,30 +4,26 @@
 /** Build related information */
 export interface BuildInfo {
   /** The operating system CPU architecture. */
-  arch: "x64";
+  arch: "x64" | "arm64";
 
-  /** The operating system platform. */
-  os: OSType;
+  /** The operating system. */
+  os: OperatingSystem;
 
   /** The arguments passed to GN during build. See `gn help buildargs`. */
   args: string;
 }
 
 /** The operating system platform. */
-export enum OSType {
-  mac = "mac",
-  win = "win",
-  linux = "linux"
-}
+export type OperatingSystem = "mac" | "win" | "linux";
 
 // 'build' is injected by rollup.config.js at compile time.
 export const build: BuildInfo = {
-  /* eslint-disable @typescript-eslint/no-explicit-any */
   // These string will be replaced by rollup
```

Comment:
If you use disable-next-line then you don't need to re-enable below.

Reply:
oops - fixed.